src/test/java/edu/hm/hafner/analysis/parser/TrivyParserTest.java

```diff
@@ -0,0 +1,31 @@
+package edu.hm.hafner.analysis.parser;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.assertions.SoftAssertions;
+
```

Comment:
Please add a simple JavaDoc to the class, see other tests.

Reply:
I do think that a comment like "Tests the class xy" will improve anything, but its your code base.

Don't get me wrong comments are important but only on places where its needed to understand whats going on. I would rather suggest do reduce commenting but choose better names for methods variables mambers and so on...